### PR TITLE
Task adjust per page

### PR DESCRIPTION
**Pull request by JamesCullen2**
Kevin - I've worked on about and portfolio section; mainly focused on structuring social media icons to work vertically in mobile and tablet but inline horizontally in desktop version - I'd really appreciate your feedback, J

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -60,18 +60,27 @@ section {
...
     width: 100%;
     height: 30vh;
 }
 
+.footer-nav-bar {
+    float: left;
+    margin: 0 auto;
+    width: 25%;
+    border: 5px white;
+    box-sizing: border-box;
```
Comment (obwansan) [inline styles.css]: would it work just having box-sizing: border-box; on the body tag?

File: styles.css [hunk 2]
```diff
@@ -96,10 +102,28 @@ section {
...
+        display: inline-block;
+        width: 100%;
+        height: 30vh;
     }
-}
+    .footer-nav-bar {
+        float: left;
+        margin: 0 auto;
+        width: 100%;
+        /*margin-top: 50px;*/
```
Comment (obwansan) [inline styles.css]: remove unused code before sending to Charlie?

File: styles.css [hunk 3]
```diff
@@ -96,10 +102,28 @@ section {
...
+        border: 5px white;
+        box-sizing: border-box;
```
Comment (obwansan) [inline styles.css]: Same question as above

File: styles.css (hunk 1 above)
Comment (JamesCullen2) [reply styles.css]: Yes ! it would ! I've deleted unnecessary box-sizing tags

File: styles.css (hunk 3 above)
Comment (JamesCullen2) [reply styles.css]: done

File: styles.css (hunk 2 above)
Comment (JamesCullen2) [reply styles.css]: removed

**Review by obwansan — APPROVED** [review]

**Pull request merged**